js/models/serverConfigMd.js

```diff
@@ -0,0 +1,123 @@
+var Backbone = require('backbone'),
```

Comment:
This is the model where local server configuration is stored. It is driven via the Server Connect modal and it persists its data in localStorage using a Backbone Local Storage adapter.